File: 58/solution.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        n = len(s)
        i = n - 1
        while i >= 0 and s[i] == ' ':
            i -= 1
        j = i
        while j >= 0 and s[j] != ' ':
            j -= 1
        return i - j

File: 58/solution.go
package _58

func lengthOfLastWord(s string) int {
	n := len(s)
	i := n - 1
	for i >= 0 && s[i] == ' ' {
		i -= 1
	}
	j := i
	for j >= 0 && s[j] != ' ' {
		j -= 1
	}
	return i - j
}
